src/detect-engine-register.c

```diff
@@ -739,6 +727,14 @@ void SigTableSetup(void)
     ScDetectSipRegister();
     ScDetectTemplateRegister();
     ScDetectLdapRegister();
+    SCDetectDNSRegister();
+    /* For lua : register these generic engines from here for now */
```

Comment:
@jasonish what is going on here for lua ?

Reply:
? I'm not really familiar with this bit. Are these registrations for Lua only?

Reply:
Apparently cf `git grep '"dns_request"'` it gets used in detect-lua.c and it is needed for the SV test about dns and lua

Reply:
I think we're looking to cleanup all this Lua stuff related to this.  I admit I'm not that familiar with how it works at this time.